html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+<form>
+  Email <br>
+  <label for="email">
+    <input type="email" id="email" name="email">
+  </label> <br>
```

Comment:
We want to avoid using `<br>` tags, it is better to wrap elements in `<div>`s instead. Also the text for the input should be within the label. For example:

```HTML
<div> 
  <label for="email">Email</label>
  <div>
    <input type="email" id="email" name="email">
  </div>
</div>
```